### ISSUE TITLE
How do I connect Bert and the network using Lambada?

### ISSUE THREAD

**Issue by fatemeh-sh264** [imgarylai/bert-embedding#43]
import os
import json
from keras.preprocessing.text import Tokenizer
from keras.preprocessing.sequence import pad_sequences
import numpy as np
import pandas as pd
from scipy import sparse
from keras.models import Sequential
import matplotlib.pyplot as plt
from bert_embedding import BertEmbedding
from keras.layers import LSTM, Activation, Dense, Dropout, Input,Lambda
 
filename1= ('/content/drive/My Drive/webtext.train.csv')
with open(filename1) as file:
  dr1=pd.read_csv(file)
train_webtext=pd.DataFrame(dr1,columns=["text"])
 
filename2= ('/content/drive/My Drive/xl-1542M-k40.train.csv')
with open(filename2) as file:
  dr2=pd.read_csv(file)
train_gen=pd.DataFrame(dr2,columns=["text"])

filename3= ('/content/drive/My Drive/webtext.valid.csv')
with open(filename3) as file:
  dr3=pd.read_csv(file)
valid_webtext=pd.DataFrame(dr3,columns=["text"])
 
filename4=('/content/drive/My Drive/xl-1542M-k40.valid.csv')
with open(filename4) as file:
  dr4=pd.read_csv(file)
valid_gen=pd.DataFrame(dr4,columns=["text"])

filename5=('/content/drive/My Drive/xl-1542M-k40.test.csv')
with open(filename5) as file:
  dr5=pd.read_csv(file)
test_webtext=pd.DataFrame(dr5,columns=["text"])

filename6=('/content/drive/My Drive/xl-1542M-k40.test.csv')
with open(filename6) as file:
  dr6=pd.read_csv(file)
test_gen=pd.DataFrame(dr6,columns=["text"])

texts = pd.concat([train_webtext,train_gen,valid_webtext,valid_gen,test_webtext,test_gen])
labels = [0]*len(train_webtext)+[0]*len(valid_webtext)+[0]*len(test_webtext)+[1]*len(train_gen)+[1]*len(valid_gen)+[1]*len(test_gen)

for line in texts:
        sentences = line.split('\n')
        bert_embedding = BertEmbedding(model='bert_24_1024_16', dataset_name='book_corpus_wiki_en_cased')
result = bert_embedding(sentences)

x_train=result[:500000]
y_train=labels[:500000]
x_valid=result[500001:510001]
y_valid=labels[500001:510001]
x_test=result[510002:520002]
y_test=labels[510002:520002]

model=Sequential()
**model.add(Lambda(, output_shape=(embed_size,))(input_text))**؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟
model.add(LSTM(128))
model.add(Dense(1, activation='sigmoid'))
model.compile(optimizer='rmsprop', loss='binary_crossentropy', metrics=['acc'])
history = model.fit(x_train, y_train ,epochs=2,batch_size=128,validation_data=(x_valid,y_valid))
